src/plugins/scriptedwizard/resources/plugins/wizard.script

```diff
@@ -289,30 +289,6 @@ function SetupProjectBuildOptions(project)
         project.AddIncludeDir(_T("$(#cb)\\include"));
         project.AddIncludeDir(_T("$(#cb)\\sdk\\wxscintilla\\include"));
         project.AddIncludeDir(_T("$(#cb)\\include\\tinyxml"));
-        if (WxVersion == 0)
-        {
-            project.AddIncludeDir(_T("$(#wx.include)"));
-            project.AddIncludeDir(_T("$(#wx.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw28u"));
-            project.AddLibDir(_T("$(#wx.lib)\\gcc_dll"));
-        }
-        else if (WxVersion == 1)
-        {
-            project.AddIncludeDir(_T("$(#wx30.include)"));
-            project.AddIncludeDir(_T("$(#wx30.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw30u"));
-            project.AddLibDir(_T("$(#wx30.lib)\\gcc_dll"));
-        }
-        else if (WxVersion == 2)
-        {
-            project.AddIncludeDir(_T("$(#wx31.include)"));
-            project.AddIncludeDir(_T("$(#wx31.lib)\\gcc_dll\\mswu"));
-
-            project.AddLinkLib(_T("wxmsw31u"));
-            project.AddLibDir(_T("$(#wx31.lib)\\gcc_dll"));
-        }
 
         project.AddLibDir(GetBaseDevelPath(WxVersion));
```

Comment:
Shouldn't this be moved per target? On linux codeblocks_wx28 and codeblocks_wx30 link to different libcodeblocks.so. Isn't this the case on windows?